project_app/components/CardBox.js

```diff
@@ -0,0 +1,65 @@
+import Grid from "@mui/material/Grid";
+import CardComponent from "./CardComponent";
+import Image from "next/image";
+import {
+  ReviewsBox,
+  LeftWaterContainer,
+  RightWaterContainer,
+  MazeContainer,
+} from "./styled";
+const cards = [1, 2];
+
+export default function CardBox() {
+  return (
+    <>
+      <ReviewsBox>
+        <MazeContainer>
+          <Image
+            src="/Maze.png"
+            fill="absolute"
+            objectFit="cover"
+            alt="picture of the background"
+          />
+        </MazeContainer>
+
+        {/* <div
+          style={{ position: "absolute", left: "0%", background: "#73D1E9" }}
+        >
+          <Image src="/left_review.png" width={20} height={20} alt="image" />
+        </div> */}
+        <LeftWaterContainer>
+          <Image
+            src="/left_review.png"
+            width={103.74}
+            height={343.85}
+            alt="image"
+          />
+        </LeftWaterContainer>
+        <Grid
```

Comment:
the water looks weird, but we can fix that later

Reply:
Copy that